docs/platform/object-storage/object-storage-ga-shortguide/index.md

```diff
@@ -0,0 +1,19 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'Shortguide that displays the general availability note.'
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2020-02-12
+modified_by:
+  name: Heather Zoppetti
+published: 2020-02-12
+title: Object Storage General Availability Notice
+keywords: []
+headless: true
+show_on_rss_feed: false
+---
+
+{{< note >}}
+[Linode Object Storage](/docs/platform/object-storage/) is now available to the general public in the Newark and Frankfurt data centers! Starting November 1, 2019, all customers with the Object Storage service enabled on their account will be billed. For more information, see our [Object Storage Pricing and Limitations](/docs/platform/object-storage/pricing-and-limitations/) guide.
```

Comment:
hey @hzoppetti, what's your opinion on getting rid of `Starting November 1, 2019, all customers with the Object Storage service enabled on their account will be billed.` in this? People have been charged for a few billing cycles now, so it feels like that's probably the expectation now

Reply:
Sounds good to me